tool action does not find new script location

_From @joejulian on August 15, 2017 15:58_

With the scripts having been moved, this now occurs:
1.0.7-stable+git.sha.d7454e05aab6b0a3078b660fb44497dfa25a853a

```
k2cli tool kubectl --config ~/.kraken/jj-r53.yaml get nodes
Using K2 config file: /home/jjulian/.kraken/jj-r53.yaml
Error response from daemon: oci runtime error: container_linux.go:262: starting container process caused "exec: \"./computed_kubectl.sh\": stat ./computed_kubectl.sh: no such file or directory"
panic: Error response from daemon: oci runtime error: container_linux.go:262: starting container process caused "exec: \"./computed_kubectl.sh\": stat ./computed_kubectl.sh: no such file or directory"
```

_Copied from original issue: samsung-cnct/k2#754_

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 NAME      := k2cli
-VERSION   := 1.0.6
+VERSION   := 1.0.8
 TYPE      := stable
 COMMIT    := $(shell git rev-parse HEAD)
 godep=GOPATH=$(shell godep path):${GOPATH}

